README.md

```diff
@@ -1,7 +1,7 @@
 # go-oidc
 
-[![GoDoc](https://godoc.org/github.com/coreos/go-oidc?status.svg)](https://godoc.org/github.com/coreos/go-oidc)
-[![Build Status](https://travis-ci.org/coreos/go-oidc.png?branch=master)](https://travis-ci.org/coreos/go-oidc)
+[![GoDoc](https://godoc.org/github.com/coreos/go-oidc?status.svg)](https://pkg.go.dev/github.com/coreos/go-oidc/v3/oidc)
```

Comment:
https://pkg.go.dev/github.com/coreos/go-oidc/v3/oidc returns a 404

Reply:
Working on that :)

Reply:
Apologies.

Reply:
Thanks for helping review! Should be up now. Having trouble with go mods, ha.

Reply:
Happy to help.

Once in a while you come across an oss project that has a great API, maintained and works as expected. Such projects illicit a feeling analogous to a neighbours cute puppy .. not really yours but can come play anytime and have a blast.